### DISCUSSION TITLE
V2.1.1 ETS6 Projekt invalid password

### DISCUSSION THREAD

**Discussion by Guido1976** [brutella/hkknx-public#154] [General]
Hallo,
ich habe ein Problem, dass ich auf der KNX Bridge auf dem Raspberry Pi mein neues ETS6 Projekt nicht importiert bekomme. Es kommt immer die Fehlermeldung zip: invalid password.
Wenn ich das alte Projekt von der ETS5 importiere geht es reibungslos. Woran kann das liegen?

Grüße Guido

[a comment by an automated account is not shown]

**Reply by Guido1976** [reply]
Ja, beides. MDT IP Router mit Secure. Passwort im Projekt und beim exportieren das Passwort eingetragen.
Ich verstehe halt nicht, warum das gleiche Projekt vor dem Import in die ETS6 noch funktioniert und danach nicht mehr.

**Reply by Guido1976** [reply]
Ich habe heute morgen nochmal mehrere Versuche gestartet und dabei festgestellt, dass auch das entfernen meines MDT Ip Secure Routers noch nicht zum Erfolg führte. Wenn ich aber dann das Passwort entfernte, funktionierte der Import.

**Comment by brutella** [comment]
Tatsächlich scheint es einen Unterschied zu geben, ob ein Projekt mit Passwortschutz unter ETS5 oder ETS6 exportiert wurde. Dem genauen Grund muss ich noch nachgehen.

**Reply by Guido1976** [reply]
Hallo,

Gibt es inzwischen Neuigkeiten zu dem Problem mit dem ETS 6 Projekt mit Passwort?

**Reply by brutella** [reply]
Laut [knx-user-forum](https://knx-user-forum.de/forum/öffentlicher-bereich/knx-eib-forum/1711442-erster-ausflug-in-die-ets6-ernüchterung?p=1713912#post1713912) kann ein exportiertes ETS 6 Projekt mit einem Passwort nicht mehr so ohne weiteres entschlüsselt werden.

**Reply by rrnetz** [reply]
Guten Morgen zusammen, gibt es hierzu schon eine "neuere" Info? Bei mir schlägt der import einer ETS6 KNX secure / Passwort geschützte Datei fehl... Danke & Gruß Stefan

**Reply by brutella** [reply]
Passwortgeschütze ETS6 Projektdateien können jetzt mit dem aktuellsten Build von Version [2.3.1](https://github.com/brutella/hkknx-public/releases/tag/2.3.1) importiert werden.

**Reply by rrnetz** [reply]
Hi, soeben erfolgreich durchgeführt. MEGA! DANKE!
